addons/binding/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/handler/UniFiClientThingHandler.java

```diff
@@ -111,15 +114,34 @@ public void initialize() {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        // mgb: only handle the command if we're ONLINE
-        if (getThing().getStatus() == ONLINE) {
```

Comment:
Why it the channel block called when the device is offline? If the device is off line it can't accept commands I would assume.

Reply:
If the client gets blocked it will be put "OFFLINE". In order to get it "unblocked" in has to be able to handle commands to Things that are offline. Later on in the binding (when API is called) it gets data from "Insights" which is the "offline"/history database of disconnected/offline clients.

Reply:
Are you sure about this? If I block a device it stays online, but the channel `CHANNEL_ONLINE` becomes off. In other words. Don't you mix up between thing status online/offline and the channel `CHANNEL_ONLINE`?
These checks are for the thing status not the status given by unifi controller about the device.

Reply:
Exactly - I've fixed this and it will be commited during the day.

Reply:
Changed now.